contracts/router/src/contract.rs

```diff
@@ -1,35 +1,25 @@
-use axelar_wasm_std::{address, killswitch, migrate_from_version, permission_control, FnExt};
+use axelar_wasm_std::{address, killswitch, permission_control, FnExt};
 #[cfg(not(feature = "library"))]
 use cosmwasm_std::entry_point;
 use cosmwasm_std::{
     to_json_binary, Addr, Binary, Deps, DepsMut, Env, MessageInfo, Response, Storage,
 };
 use router_api::error::Error;
 
-use crate::contract::migrations::v1_1_1;
 use crate::events::RouterInstantiated;
-use crate::msg::{ExecuteMsg, InstantiateMsg, MigrateMsg, QueryMsg};
+use crate::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
 use crate::state;
 use crate::state::{load_chain_by_gateway, load_config, Config};
 
 mod execute;
-mod migrations;
+pub(crate) mod migrations;
```

Comment:
why does this need to be pub(crate)?

Reply:
can it be restricted to this?
```suggestion
mod migrations;

pub(crate) use migrations::MigrateMsg
```

Reply:
it needs to be pub so that it can be re-exported in `msg.rs`